src/workers/guessWikidata.ts

```diff
@@ -180,6 +209,20 @@ const guessWikidata = new DiscordWorker<GuessWikidataJob>(
       )
     }
 
+    const checkIfWikidataExistInProduction = await fetch(apiConfig.prod_base_url + '/companies/' + wikidataForApproval.node, {method: 'GET', headers: {'Content-Type': 'application/json'}});    
```

Comment:
could have a more suitable variable name

Reply:
also, we should maybe add some logging (`job.log`) in here, and error handling using try catch